com.ekitabu.eKitabu.json

```diff
@@ -0,0 +1,53 @@
+{
+    "app-id": "com.ekitabu.eKitabu",
+    "base": "io.atom.electron.BaseApp",
+    "base-version": "20.08",
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "22.08",
+    "sdk": "org.freedesktop.Sdk",
+    "command": "ekitabu",
+    "separate-locales": false,
+    "finish-args": [
+        "--socket=pulseaudio",
+        "--share=network",
+        "--filesystem=xdg-documents:ro",
+        "--filesystem=xdg-videos:ro",
+        "--filesystem=xdg-download",
+        "--talk-name=com.canonical.AppMenu.Registrar"
```

Comment:
it's an electron app and doesn't even have any permission to display. did you even test this?